sopa/cli/segmentation.py

```diff
@@ -27,6 +28,10 @@ def cellpose(
         0.2,
         help="Parameter for skimage.exposure.equalize_adapthist (applied before running cellpose)",
     ),
+    clahe_kernel_size: Optional[Union[int, Iterable[int]]] = typer.Option(
```

Comment:
Unfortunately, `typer` (the CLI tool) currently doesn't support Union types. We will not be able to pass an `Iterable[int]` to the CLI...Only the API will work with `int | Iterable[int] | None`

Reply:
okay, what should the typing be? just int?

Reply:
Yes, just int (and `None` as a default value)